client/src/components/layout/ParkList.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect } from "react"
+import ParkTile from "./ParkTile.js"
+
+const ParkList = (props) => {
+
+  const [park, setPark] = useState([])
+
+
+  const getParks = async () => {
+    try {
+      const response = await fetch("/api/v1/parks")
+
+      if(!response.ok){
+        const errorMessage = `${response.status} (${response.statusText})`
+        const error = new Error(errorMessage)
+        throw(error)
+      }
+      const body = await response.json()
+
+      setPark(body.parks)
+
+    } catch (error) {
+      console.log(error)
+      console.error(`Error in fetch: ${error.message}`)
+    }
+  }
+  useEffect(() => {
+    getParks()
+  }, [])
+
+  const parksListItems = park.map(parksItem => {
+
+    return(
+    <>
+      <ParkTile key={parksItem.id}
+        {...parksItem}
+      />
+    </>
+    )
```

Comment:
If there are errors in the terminal or console, you should take a look at resolving those before putting incomplete code up for review, consult your other team members or the QQ if you're unsure! 
The message of `Check the render method of `ParkList`` `in Fragment (created by ParkList)` is because the top-level item being returned from this `map` is the fragment. When we apply the `key` property, it needs to be added to the top most element being returned, which in this case is the Fragment. 
My recommendation here is to just remove the fragment `<>` all together because it doesn't provide us any value. We would use a fragment if we were returning multiple elements like a `h1` and a `p`, but don't want to wrap them both in a `div` because it would add clutter to the DOM.
Using the fragment here applies it to each park we iterate over to create a ParkTile. If we wanted to apply some styling to each park tile, the appropriate place would be in the `ParkTile` itself